rosdep/base.yaml

```diff
@@ -69,6 +69,9 @@ apr:
   macports: [apr, apr-util]
   opensuse: [libapr1, libapr-util1]
   rhel: [apr-devel, apr-util]
+  slackware:
+    slackpkg:
```

Comment:
Is slackpkg not the default package manager? I see three types of rules default, slackpkg, and pip. Clearly the pip ones needs to be long format, but does the slackpkg also?